parachains/runtimes/collectives/collectives-polkadot/src/fellowship/mod.rs

```diff
@@ -108,35 +109,93 @@ morph_types! {
+	// Demotion is by any of:
+	// - Root can demote arbitrarily.
 	// - the FellowshipAdmin origin (i.e. token holder referendum);
-	// - a vote by the rank *above* the new rank.
-	type PromoteOrigin = EitherOf<
+	// - a vote by the rank two above the current rank.
+	type DemoteOrigin = EitherOf<
 		frame_system::EnsureRootWithSuccess<Self::AccountId, ConstU16<65535>>,
 		EitherOf<
 			MapSuccess<
 				EnsureXcm<IsVoiceOfBody<GovernanceLocation, FellowshipAdminBodyId>>,
 				Replace<ConstU16<9>>,
 			>,
-			TryMapSuccess<EnsureFellowship, CheckedReduceBy<ConstU16<1>>>,
+			TryMapSuccess<EnsureFellowship, CheckedReduceBy<ConstU16<2>>>,
 		>,
 	>;
-	// Demotion is by any of:
-	// - Root can demote arbitrarily.
+	type Polls = FellowshipReferenda;
+	type MinRankOfClass = sp_runtime::traits::Identity;
+	type VoteWeight = pallet_ranked_collective::Geometric;
+}
+
+pub type FellowshipCoreInstance = pallet_core_fellowship::Instance1;
+
+impl pallet_core_fellowship::Config<FellowshipCoreInstance> for Runtime {
+	type WeightInfo = weights::pallet_core_fellowship::WeightInfo<Runtime>;
+	type RuntimeEvent = RuntimeEvent;
+	type Members = pallet_ranked_collective::Pallet<Runtime, FellowshipCollectiveInstance>;
+	type Balance = Balance;
+	// Parameters are set by any of:
+	// - Root;
+	// - a vote of the rank 3 or above.
+	type ParamsOrigin = Fellows;
+	// Induction is by any of:
+	// - Root;
+	// - a member of the rank 1 or above.
+	type InductOrigin = EnsureFellowship;
+	// Approval of a member's current rank is by any of:
+	// - Root;
 	// - the FellowshipAdmin origin (i.e. token holder referendum);
-	// - a vote by the rank two above the current rank.
-	type DemoteOrigin = EitherOf<
+	// - a vote by the rank above or equal to the current rank.
+	type ApproveOrigin = EitherOf<
+		MapSuccess<
+			EnsureXcm<IsVoiceOfBody<GovernanceLocation, FellowshipAdminBodyId>>,
+			Replace<ConstU16<9>>,
+		>,
+		EnsureFellowship,
+	>;
+	// Promotion is by any of:
+	// - Root can promote arbitrarily.
+	// - the FellowshipAdmin origin (i.e. token holder referendum);
+	// - a vote by the rank *above* the new rank.
+	type PromoteOrigin = EitherOf<
 		frame_system::EnsureRootWithSuccess<Self::AccountId, ConstU16<65535>>,
 		EitherOf<
 			MapSuccess<
 				EnsureXcm<IsVoiceOfBody<GovernanceLocation, FellowshipAdminBodyId>>,
 				Replace<ConstU16<9>>,
 			>,
-			TryMapSuccess<EnsureFellowship, CheckedReduceBy<ConstU16<2>>>,
+			TryMapSuccess<EnsureFellowship, CheckedReduceBy<ConstU16<1>>>,
 		>,
 	>;
-	type Polls = FellowshipReferenda;
-	type MinRankOfClass = sp_runtime::traits::Identity;
-	type VoteWeight = pallet_ranked_collective::Geometric;
+	type EvidenceSize = ConstU32<1024>;
+}
+
+parameter_types! {
+	pub FellowshipAccount: AccountId = constants::account::FELLOWSHIP_PALLET_ID.into_account_truncating();
+	pub const RegistrationPeriod: BlockNumber = 15 * DAYS;
+	pub const PayoutPeriod: BlockNumber = 15 * DAYS;
```

Comment:
The one cycle is 30 days, for comparison the members activity checks should be done every 90 days for members below the rank Fellows (according to the Fellowship manifesto).